apps/opik-backend/src/main/resources/liquibase/db-app-state/migrations/000004_add_prompt_library_tables.sql

```diff
@@ -0,0 +1,33 @@
+--liquibase formatted sql
+--changeset thiagohora:add_prompt_library_tables
+
+CREATE TABLE prompts (
+    id CHAR(36) NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    description VARCHAR(255),
+    created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
+    created_by VARCHAR(100) NOT NULL DEFAULT 'admin',
+    last_updated_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),
+    last_updated_by VARCHAR(100) NOT NULL DEFAULT 'admin',
+    workspace_id VARCHAR(150) NOT NULL,
+    CONSTRAINT `prompts_pk` PRIMARY KEY (id),
+    CONSTRAINT `prompts_workspace_id_name_uk` UNIQUE (workspace_id, name)
```

Comment:
It might be more efficient to reorder the fields, placing the `name` field first. This would allow us to use the index for faster search performance.

Reply:
I agree with you, but the (name, workspace_id) index will primarily optimize queries that search by name. Meanwhile, (workspace_id, name) would be more flexible, as it can support queries that use workspace_id alone or combined with the name (leftmost prefix rule).